lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -86,6 +98,100 @@ sub execute {
     return 1;
 }
 
+sub get_bam_from_model {
+    my $self = shift;
+    my $model = Genome::Model->get(id => $self->model);
+
+    my $build = $model->last_succeeded_build;
+    unless ($build) {
+        die "Couldn't find a recent successful build for model: ",
```

Comment:
Another thought if this makes it way towards a pipeline:  Our automated failure classifier works better if these are `$self->error_message()`s.  Many places in our code do `die $self->error_message('...')` (as we don't currently have a separate `fatal_message`).

(Unlike `die`, they interpret multiple args like `sprintf` does, so you'd need to add a `%s`.)